imageprocessor/templates/view_my_pictures.html

```diff
@@ -0,0 +1,38 @@
+{% extends "base.html" %}
+{% block body %}
+<style>
+    h2 { margin: 20px; }
+    a { display: block; margin: 20px; }
+
+</style>
+{% block css %}
+    <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.19/css/jquery.dataTables.css">
+{% endblock %}
+{% block javascript %}
+    <script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.10.19/js/jquery.dataTables.js"></script>
+{% endblock %}
+<div>
+    <h2>My Images</h2>
+    <table id="photo-table" class="display">
+    <thead>
+        <tr>
+            <th>Photos</th>
+        </tr>
+    </thead>
+    <tbody>
+    {% for picture in my_pictures %}
+        <tr>
+            <td>{% for tag in picture.get_tags %}<a href = "{{picture.url}}">{{tag}}</a> {% endfor %}</td>
```

Comment:
If you want to render images, you might want to try loading cloudinary and following the pattern I set up in the view_tagged_photos page

Reply:
Oh, by the way, since an image can have multiple tags, it ends up being a litter weird to iterate over all the tags in the photo. It ends up just repeating the image or url once for each tag

Reply:
added